repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -578,18 +577,24 @@ export const clickRandomElementInTable = (tableName: string, elementId: string):
 };
 
 export const completeOperatorSearch = (isMultiService: boolean): void => {
-
     clickElementById('test-radio');
     continueButtonClick();
-
 };
 
 export const addFlatFareProductIfNotPresent = (): void => {
     getHomePage();
     clickElementById('manage-fares-link');
     clickElementByText('Other products');
-    getElementByClass('govuk-table').then((table) => {
-        if (table.find('tr').length === 1) {
+    const hasFlatFare: JQuery<HTMLElement>[] = [];
```

Comment:
names like 'hasFlatFare' would usually mean its a boolean (because something either has flat fare or does not have flat fare). If we want to use a boolean, could we just set this to false, and then change it to true when we need to?

Or if we need it to be an array of Elements, can we use a name like flatFareProducts or something more suitable?